File: xianggole-sellergoods-interface/src/main/java/com/xianggole/sellergoods/service/BrandService.java
package com.xianggole.sellergoods.service;

import java.util.List;

import com.xianggole.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	
	public List<TbBrand> findAll();
	
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 分页查询
	 * @param brand
	 */
	public PageResult findPage(TbBrand brand ,int pageNum,int pageSize);
	
	
	public void add(TbBrand brand);
	
	/**
	 * 查找一个brand
	 */
	
	public TbBrand findOne(Long id);
	
	/**
	 * 修改brand
	 */
	
	public void update(TbBrand brand);
	
	/**
	 * 删除
	 */
	public void delete(Long[] ids);

}

File: xianggole-manager-web/src/main/webapp/js/service/brandService.js
	app.service('brandService',function($http){
		this.findAll=function(){
			return $http.get('../brand/findAll.do');
		}
		this.findOne=function(id){
			return $http.get('../brand/findOne.do?id='+id);
		}
		this.add=function(entity){
			return $http.post('../brand/add.do',entity);
		}
		this.update=function(entity){
			return $http.post('../brand/update.do',entity);
		}
		this.delet =function(selectIds){
			return $http.get('../brand/delete.do?ids='+selectIds);
		} 
		this.search = function(page,size,searchEntity){
			return $http.post('../brand/search.do?page='+page+'&size='+size,searchEntity);
		}
	});